.github/workflows/docs.yml

```diff
@@ -2,7 +2,7 @@ name: docs
 on:
   push:
     branches: [main]
-    paths: ['src/**/*.rs', 'Cargo.toml', '.github/workflows/docs.yml']
+    paths: ['crates/**/*.rs', '**/Cargo.toml', '.github/workflows/docs.yml']
```

Comment:
I noticed #5 didn't trigger the docs workflow while it should have

Reply:
Right!